docs/tutorial/devtools-extension.md

```diff
@@ -8,6 +8,8 @@ the `BrowserWindow.addDevToolsExtension` API to load them. The loaded extensions
 will be remembered so you don't need to call the API every time when creating
 a window.
 
+## NOTE: React DevTools does not work, follow the issue here https://github.com/atom/electron/issues/915
```

Comment:
But you should not be able to miss it, it is not working so maybe the react example should be removed instead?